sdk/core/System.ClientModel/api/System.ClientModel.net8.0.cs

```diff
@@ -223,13 +233,28 @@ public MessageLoggingPolicy(System.ClientModel.Primitives.ClientLoggingOptions?
         public sealed override void Process(System.ClientModel.Primitives.PipelineMessage message, System.Collections.Generic.IReadOnlyList<System.ClientModel.Primitives.PipelinePolicy> pipeline, int currentIndex) { }
         public sealed override System.Threading.Tasks.ValueTask ProcessAsync(System.ClientModel.Primitives.PipelineMessage message, System.Collections.Generic.IReadOnlyList<System.ClientModel.Primitives.PipelinePolicy> pipeline, int currentIndex) { throw null; }
     }
+    public abstract partial class ModelInfo
```

Comment:
We should probably come up with a better name for this. Maybe ModelInitializer?

Reply:
PersistableModelInfo?

Reply:
My issue was more with "Info".

Reply:
renamed to ModelBuilder per review fb